tutorials/physics/using_kinematic_body_2d.rst

```diff
@@ -170,7 +177,42 @@ Attach a script to the KinematicBody2D and add the following code:
 
  .. code-tab:: csharp
 
-    Coming soon
+    using Godot;
+    using System;
+
+    public class KBExample : KinematicBody2D
+    { 
+        public int Speed = 250;
+        private Vector2 velocity = new Vector2();
```

Comment:
Could you change it to `_velocity`? We recommend all private fields to be in camelCase with `_` prefix now.